src/pages/Register/Register.jsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+// Hooks
+import { useFormik } from "formik";
+// Styles
+import "./Register.scss";
+// Components
+import { Typography, Button, Grid, TextField, Container, FormHelperText } from "@material-ui/core";
+
+import RegisterSchema from "./RegisterSchema";
+
+const Register = () => {
+  const formik = useFormik({
+    initialValues: {
+      name: "",
+      email: "",
+      password: "",
+    },
+    initialErrors: {
+      name: false,
+      email: false,
+      password: false,
+    },
+    initialTouched: {
+      name: false,
+      email: false,
+      password: false,
+    },
+    validationSchema: RegisterSchema,
```

Comment:
Agregar `validateOnChange: false` para que no esté validando apenas ingresás algo

```suggestion
    validationSchema: RegisterSchema,
    validateOnChange: false,
```